src/indexing.jl

```diff
@@ -48,16 +48,16 @@ Base.eachindex(A::AxisArray) = eachindex(A.data)
 This internal function determines the new set of axes that are constructed upon
 indexing with I.
 """
-reaxis(A::AxisArray, I::Idx...) = _reaxis(make_axes_match(axes(A), I), I)
+reaxis(A::AxisArray, copy::Val, I::Idx...) = _reaxis(make_axes_match(axes(A), I), copy, I)
```

Comment:
Best to change the docstring too if you're changing the signature.

Reply:
Fixed.